Mathlib/Algebra/Order/Floor.lean

```diff
@@ -1395,6 +1396,14 @@ theorem abs_sub_round_div_natCast_eq {m n : ℕ} :
   rw [abs_sub_round_eq_min, Nat.cast_min, ← min_div_div_right hn'.le,
     fract_div_natCast_eq_div_natCast_mod, Nat.cast_sub (m.mod_lt hn).le, sub_div, div_self hn'.ne']
 
+theorem round_half_bot (x : α) : x - 1 / 2 < round x := by
+  rw [round_eq x, show x - 1 / 2 = x + 1 / 2 - 1 by ring]
+  exact Int.sub_one_lt_floor (x + 1 / 2)
+
+theorem round_half_top (x : α) : (round x : α) ≤ x + 1 / 2 := by
```

Comment:
I will let you find the correct name of this one as an exercise

Reply:
I renamed it to `round_le_add_half`. Is that correct?

Reply:
Yes!